src/modules/emanation/emanation.ts

```diff
@@ -163,7 +164,7 @@ async function spellSheetRenderWrapper(this: SpellSheetPF2e, wrapped, ...args) {
 	try {
 		if (MODULE.settings.emanationAutomation) await spellSheetRenderInner(this, $html)
 	} catch (e) {
-		ui.notifications.error("Could not insert emanation automation into spell sheet.")
+		ui.notifications.error(translate("emanation.spellsheet-error"))
```

Comment:
This shouldn't be translated either. Another error I need to be able to read if it happens. The ui.notifactions.error is just here because this silently broke before.
```suggestion
		ui.notifications.error("Could not insert emanation automation into spell sheet. Check console for error.")
```